cmd/ui/src/views/Explore/ExploreSearch/CommonSearches.tsx

```diff
@@ -39,7 +39,10 @@ const useStyles = makeStyles(() => ({
     tab: {
         height: '35px',
         minHeight: '35px',
-        color: 'black',
+        color: theme.palette.color.primary,
+        '&.Mui-selected': {
```

Comment:
Should this be an override?

Reply:
confirmed that it targets tab color of MUI Tabs component and an override would better support any future uses of tabs